usl_pipeline/cloud_functions/main_test.py

```diff
@@ -86,8 +96,9 @@ def test_build_feature_matrix_flood(mock_storage_client, mock_firestore_client,
         [
             mock.call().bucket("bucket"),
             mock.call().bucket().blob("study_area/name.tar"),
-            mock.call().bucket("climateiq-study-area-feature-chunks"),
-            mock.call().bucket().blob("study_area/name.npy"),
+            # I don't understand why the following calls are not registered in the mock!
```

Comment:
They do seem to be. When I un-comment the two lines bellow you commented out, the tests pass.

Reply:
Wow, it works now for me too... I can't explain why it didn't last week.